chain/client/src/view_client.rs

```diff
@@ -43,7 +43,7 @@ impl ViewClientActor {
             Ok(result) => Ok(result.clone().into()),
             Err(err) => match err.kind() {
                 ErrorKind::DBNotFoundErr(_) => {
-                    Ok(ExecutionOutcome { status: ExecutionStatus::Pending, ..Default::default() }
+                    Ok(ExecutionOutcome { status: ExecutionStatus::Unknown, ..Default::default() }
```

Comment:
Q: is this the only code path lead to `Unknown`? What does DBNotFoundErr means?

Reply:
It's not found in the DB, which is likely means the transaction/receipt was not yet processed or doesn't exist at all, e.g. fake hash